packages/vats/src/core/basic-behaviors.js

```diff
@@ -83,6 +89,166 @@ export const makeVatsFromBundles = async ({
 };
 harden(makeVatsFromBundles);
 
+/** @param {BootstrapPowers} powers */
+export const produceStartUpgradeable = async ({
+  consume: { zoe },
+  produce, // startUpgradeable
+}) => {
+  /** @type {StartUpgradeable} */
+  const startUpgradeable = async ({
```

Comment:
I don't think we've discussed this before, so not a blocker, but I really think the spelling without the `e`  is better:
```suggestion
  /** @type {StartUpgradable} */
  const startUpgradable = async ({
```
One less character and more prevelant: https://books.google.com/ngrams/graph?content=upgradeable%2Cupgradable&year_start=1800&year_end=2019&corpus=en-2019&smoothing=3